configuration/src/agent/signer.rs

```diff
@@ -12,10 +40,12 @@ pub enum SignerConf {
     /// An AWS signer. Note that AWS credentials must be inserted into the env
     /// separately.
     Aws {
-        /// The UUID identifying the AWS KMS Key
-        id: String, // change to no _ so we can set by env
-        /// The AWS region
-        region: String,
+        /// An AWS identifier for the key. This may be
+        /// 1. Its UUID
+        /// 2. Its ARN
+        /// 3. A key alias
+        /// 4. A key alias's ARN
+        id: String,
```

Comment:
Was not aware you could pass in different types of IDs when instantiating AWS signer. Could you comment in reference to documentation that describes this behavior?